public/gameInfo.php

```diff
@@ -864,6 +865,8 @@ function ResetProgress() {
                     echo "<div class='text-right'><button class='btn'>Submit</button></div>";
                     echo "<label for='game_release'>First Released</label><input type='text' name='release' id='game_release' value='" . attributeEscape($released) . "' class='w-full'>";
                     echo "<div class='text-right'><button class='btn'>Submit</button></div>";
+                    echo "<label for='guide_url'>Guide URL</label><input type='url' name='guide_url' id='guide_url' value='" . attributeEscape($guideURL) . "' class='w-full'>";
+                    echo "<div class='text-right'><button class='btn'>Submit</button></div>";
```

Comment:
Recommend moving this down near Forum Topic to be more consistent with the page layout

Reply:
Done